applications/display/display-features/org.csstudio.display.pvtable.feature/pom.xml

```diff
@@ -10,4 +10,13 @@
   <artifactId>org.csstudio.display.pvtable.feature</artifactId>
   <version>4.1.0-SNAPSHOT</version>
   <packaging>eclipse-feature</packaging>
+  
+  
+  <dependencies>
+  	<dependency>
+  		<groupId>org.apache.poi</groupId>
+  		<artifactId>poi</artifactId>
+  		<version>3.13</version>
+  	</dependency>
```

Comment:
could you move the dependencies to the manifest now that they are in the maven-osgi-bundles p2?